hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java

```diff
@@ -468,9 +468,14 @@ private boolean deleteAction(Action<Boolean> deletion, String type, Path dir) {
       LOG.debug("Couldn't delete '{}' yet because it isn't empty w/exception.", dir, exception);
       deleted = false;
     } catch (IOException ioe) {
-      LOG.info("Could not delete {} under {}. might be transient; we'll retry. if it keeps "
-          + "happening, use following exception when asking on mailing list.",
-        type, dir, ioe);
+      if (LOG.isTraceEnabled()) {
+        LOG.info("Could not delete {} under {}; will retry. If it keeps happening, " +
```

Comment:
should this be at `LOG.trace` ?

Reply:
You are right (though I suppose it doesn't matter much because of the if-check.. but let me fix. Thanks for review.